### ISSUE TITLE
`--compare` on a just created directory shows differences

### ISSUE THREAD

**Issue by nomeata** [ddssff/cabal-debian#23]
This is really minor, but maybe it would be a good sanity check if
```
$ cabal-debian
```
followed by
```
$ cabal-debian --compare
```
would show no differences.

Currently (with `--official` passed in both cases), I get two unimportant differences (order of field, and an empty line):

```
Debianization (dry run):
--- old/debian/control
+++ new/debian/control
@@
 Standards-Version: 3.9.5
 Homepage: http://hackage.haskell.org/package/mtl
-Vcs-Browser: http://darcs.debian.org/cgi-bin/darcsweb.cgi?r=pkg-haskell/haskell-mtl
-Vcs-Darcs: http://darcs.debian.org/pkg-haskell/haskell-mtl
 X-Description: Monad classes, using functional dependencies
  Monad classes using functional dependencies, with instances
@@
  by Mark P Jones, in /Advanced School of Functional Programming/, 1995
  (<http://web.cecs.pdx.edu/~mpj/pubs/springschool.html>).
+Vcs-Browser: http://darcs.debian.org/cgi-bin/darcsweb.cgi?r=pkg-haskell/haskell-mtl
+Vcs-Darcs: http://darcs.debian.org/pkg-haskell/haskell-mtl
 
 Package: libghc-mtl-dev
--- old/debian/rules
+++ new/debian/rules
@@
 
 
-

```

**Comment by ddssff** [comment]
The --compare option, by the way, is one good reason to keep the ability to both write and read debianizations.

**Comment by ddssff** [comment]
Version 4.19.2 partly resolves this issue.

**Comment by ddssff** [comment]
Fixed in version 4.19.3.
